macros/snowplow_media_player_base_events_this_run/media_session_field.sql

```diff
@@ -0,0 +1,9 @@
+{% macro media_session_field(property) %}
+    coalesce(
```

Comment:
In case `var("snowplow__enable_media_session")` is disabled it still tries to execute `coalesce(null)` which fails as it needs two arguments. I guess the same happens for all similar variable dependent macro defined fields.

Reply:
Good catch! I only checked on postgres, which allows coalesce with one argument, have changed all macros to always have more arguments in coalesce.

Reply:
Thanks! Just tried to run it again and noticed the `media_ad_field` needs updating as well, please.

Reply:
Good catch again :D Have fixed `media_ad_field` too.